cmd/argocd/commands/app_actions.go

```diff
@@ -181,3 +183,21 @@ func NewApplicationResourceActionsRunCommand(clientOpts *argocdclient.ClientOpti
 	}
 	return command
 }
+
+func getActionableResourcesForApplication(appIf applicationpkg.ApplicationServiceClient, ctx context.Context, appName *string) ([]*v1alpha1.ResourceDiff, error) {
+	resources, err := appIf.ManagedResources(ctx, &applicationpkg.ResourcesQuery{ApplicationName: appName})
+	if err != nil {
+		return resources.Items, err
+	}
+	app, err := appIf.Get(ctx, &applicationpkg.ApplicationQuery{Name: appName})
+	if err != nil {
+		return resources.Items, err
+	}
+	appGVK := app.GroupVersionKind()
+	return append(resources.Items, &v1alpha1.ResourceDiff{
+		Group:     appGVK.Group,
+		Kind:      appGVK.Kind,
+		Namespace: app.Namespace,
+		Name:      *appName,
```

Comment:
This resource diff appears to be filtered out by `util.FilterResources`.  I believe this is because its [LiveState](https://github.com/scalen/argo-cd/blob/5104816ca4563400ceb04f0619b8cf86aa44f3a1/pkg/apis/application/v1alpha1/types.go#L1196) is the empty string, which [converts to `nil`](https://github.com/argoproj/argo-cd/blob/5104816ca4563400ceb04f0619b8cf86aa44f3a1/pkg/apis/application/v1alpha1/types.go#L2475-L2477), which [gets filtered out](https://github.com/argoproj/argo-cd/blob/5104816ca4563400ceb04f0619b8cf86aa44f3a1/cmd/util/app.go#L672-L675).